### ISSUE TITLE
[no-commented-out-code] Property declaration not detected

### ISSUE THREAD

**Issue by lhoosconciso** [cartant/eslint-plugin-etc#70]
The rule `no-commented-out-code` doesn't detect this comment:

```typescript
export type MyType = {
  a: string;
  //  b: string;
  c: string;
};
```

This looks like kind of the opposite of #48, which I was not able to reproduce. That issue is still open, but has anything been done about it? After all, a todo comment kinda looks like a property declaration, so I can see how a fix for that could be the cause for this issue.
